syncd/tests/TestSyncdNvdaBf.cpp

```diff
@@ -0,0 +1,1340 @@
+        SAI_ENI_ATTR_INBOUND_V6_STAGE4_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_INBOUND_V6_STAGE5_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_OUTBOUND_V4_STAGE1_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_OUTBOUND_V4_STAGE2_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_OUTBOUND_V4_STAGE3_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_OUTBOUND_V4_STAGE4_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_OUTBOUND_V4_STAGE5_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_OUTBOUND_V6_STAGE1_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_OUTBOUND_V6_STAGE2_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_OUTBOUND_V6_STAGE3_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_OUTBOUND_V6_STAGE4_DASH_ACL_GROUP_ID,
+        SAI_ENI_ATTR_OUTBOUND_V6_STAGE5_DASH_ACL_GROUP_ID,
+    };
+
+    std::vector<sai_attribute_t> attrs;
+    std::vector<sai_object_id_t> acl_groups, acl_groups_new;
+    for (auto at : acl_attrs) {
+        sai_object_id_t acl_group;
+        EXPECT_EQ(SAI_STATUS_SUCCESS, m_sairedis->create((sai_object_type_t)SAI_OBJECT_TYPE_DASH_ACL_GROUP, &acl_group, m_switchId, 0, nullptr));
+        acl_groups.push_back(acl_group);
+
+        attr.id = at;
+        attr.value.oid = acl_group;
+        attrs.push_back(attr);
+
+        EXPECT_EQ(SAI_STATUS_SUCCESS, m_sairedis->create((sai_object_type_t)SAI_OBJECT_TYPE_DASH_ACL_GROUP, &acl_group, m_switchId, 0, nullptr));
+        acl_groups_new.push_back(acl_group);
+    }
+
+    sai_object_id_t eni;
+    EXPECT_EQ(SAI_STATUS_SUCCESS, m_sairedis->create((sai_object_type_t)SAI_OBJECT_TYPE_ENI, &eni, m_switchId, (uint32_t)attrs.size(), attrs.data()));
+
+    EXPECT_EQ(SAI_STATUS_SUCCESS, m_sairedis->get((sai_object_type_t)SAI_OBJECT_TYPE_ENI, eni, (uint32_t)attrs.size(), attrs.data()));
+    for (size_t i = 0; i < attrs.size(); i++) {
+        attr.id = attrs[i].id;
+        attr.value.oid = acl_groups_new[i];
+        EXPECT_EQ(SAI_STATUS_SUCCESS, m_sairedis->set((sai_object_type_t)SAI_OBJECT_TYPE_ENI, eni, &attr));
+    }
+
+    EXPECT_EQ(SAI_STATUS_SUCCESS, m_sairedis->get((sai_object_type_t)SAI_OBJECT_TYPE_ENI, eni, (uint32_t)attrs.size(), attrs.data()));
+
+    EXPECT_EQ(SAI_STATUS_SUCCESS, m_sairedis->remove((sai_object_type_t)SAI_OBJECT_TYPE_ENI, eni));
+}
+
+TEST_F(SyncdNvdaBfTest, dashVip)
+{
+    sai_attribute_t attr;
+
+    std::vector<sai_attribute_t> attrs;
+
+    attr.id = SAI_VIP_ENTRY_ATTR_ACTION;
+    attr.value.s32 = SAI_VIP_ENTRY_ACTION_ACCEPT;
+    attrs.push_back(attr);
+
+    sai_ip_address_t vip_addr;
+    vip_addr.addr_family = SAI_IP_ADDR_FAMILY_IPV4;
+    inet_pton(AF_INET, "192.168.0.1", &vip_addr.addr.ip4);
+
+    sai_vip_entry_t vip = { .switch_id = m_switchId, .vip = vip_addr };
+    EXPECT_EQ(SAI_STATUS_SUCCESS, m_sairedis->create(&vip, (uint32_t)attrs.size(), attrs.data()));
+
+    EXPECT_EQ(SAI_STATUS_SUCCESS, m_sairedis->get(&vip, (uint32_t)attrs.size(), attrs.data()));
+    EXPECT_EQ(attrs[0].value.s32, SAI_VIP_ENTRY_ACTION_ACCEPT);
+
+    attr.id = SAI_ENI_ETHER_ADDRESS_MAP_ENTRY_ATTR_ENI_ID;
```

Comment:
should be SAI_VIP_ENTRY_ATTR_ACTION;, since it is failing on new SAI submodule, i fixed it here: https://github.com/sonic-net/sonic-sairedis/pull/1388/files#diff-f8d332bd6741919f169fe816585ee089a1c291e290f76fb5f3954ea5796eb9d1R581